website/docs/components/rich-tooltip/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,215 @@
+
+More information about this success criteria can be found in the [accessibility](?tab=accessibility) section.
+
+!!! Dont
+
+Don’t use a [Badge](/components/badge) or [BadgeCount](/components/badge-count) as the toggle for the Rich Tooltip; Badges are intentionally non-interactive elements and don't visually communicate that content is hidden behind a `hover`, `click`, or `focus` interaction.
+
+![Example of using the Rich Tooltip with a badge](/assets/components/rich-tooltip/rich-tooltip-trigger-badge-dont.png)
+!!!
+
+!!! Do
+
+Instead, place the InfoText component inline with the Badge to communicate the existence of a Rich Tooltip.
+
+![Example of pairing the default toggle with a Badge](/assets/components/rich-tooltip/rich-tooltip-trigger-badge-do.png)
+!!!
+
+!!! Dont
+
+Don’t toggle a Rich Tooltip from a form element like a [Text Input](/components/form/text-input), [Select](/components/form/select), or [Textarea](/components/form/textarea).
+
+![Example of toggling a Rich Tooltip from a form element](/assets/components/rich-tooltip/rich-tooltip-trigger-form-input-dont.png)
+!!!
+
+!!! Do
+
+Instead, use the [Label](/components/form/primitives#formlabel) and [HelperText](/components/form/primitives#helpertext) primitives to communicate details about the field. If _absolutely_ necessary to provide more details about a form element, use the InfoText inline within [Helper Text](/components/form/primitives#formhelpertext).
+
+![Example within Helper Text](/assets/components/rich-tooltip/rich-tooltip-trigger-helper-text-do.png)
+
+However, interactive elements in HelperText will not be read out as interactive to users with screen readers, only the text will be read. More details on this recommendation can be found in the [Helper Text documentation](/components/form/primitives?tab=code#formhelpertext-1).
+!!!
+
+## Content
+
+We are not prescriptive about what type of content a Rich Tooltip can contain; consider these common examples when determining whether the Rich Tooltip is appropriate for your use case.
+
+<!-- Ensure that we have alignment for all of these scenarios -->
+
+### Text with links
+
+This is a common use case for a Rich Tooltip due to the lack of support for interactive content in the standard [Tooltip](/components/tooltip).
+
+![Link within text in the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-content-links.png)
+
+### Structured content
+
+The Rich Tooltip supports more structure and layout methods to better organize the content hierarchically.
+
+![Structured content within the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-content-structure.png)
+
+### Key-value pairs
+
+Use the Rich Tooltip to display supplemental key-value pairs or metadata related to an element.
+
+![Key-value pairs within the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-content-key-value-pair.png)
```

Comment:
[nit] Can the examples use consistent styling so we can subtly nudge folks in one direction or the other?